common/src/main/java/com/google/udmi/util/GeneralUtils.java

```diff
@@ -54,7 +54,7 @@ public static String optionsString(Object target) {
           continue;
         }
 
-        if (field.get(target) != null && Boolean.TRUE.equals(field.get(target))) {
+        if (field.get(target) != null && isTrue(field.get(target))) {
```

Comment:
I don't understand what you mean... ?

Reply:
Could you move      field.get(target)!=null  check into   isTrue   and only return  true if it's not null and also is True?

Reply:
Done. Or rather, I thought about it more and the precheck should actually be "instanceof Boolean" since False isn't the same as not-True for this particular case.